### PR TITLE
Release to Maven Central since 0.4.0

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,10 +32,8 @@ dependencies {
         exclude group: 'com.fasterxml.jackson.datatype', module: 'jackson-datatype-jdk8'
         exclude group: 'javax.validation', module: 'validation-api'
     }
-    compile 'org.embulk:embulk-util-retryhelper:0.8.1'
-    compile("org.embulk:embulk-util-json:0.1.0") {
-        exclude group: 'org.msgpack', module: 'msgpack-core'
-    }
+    compile 'org.embulk:embulk-util-retryhelper:0.8.2'
+    compile "org.embulk:embulk-util-json:0.1.1"
```
Comment (dmikurube) [inline build.gradle:47]: They have newer versions. Especially, `embulk-util-json:0.1.1` no longer depends on `msgpack-core` directly.

File: build.gradle [hunk 2]
```diff
@@ -116,23 +203,17 @@ task checkstyle(type: Checkstyle) {
 }
 
 gemPush {
-    host = "https://treasuredata.jfrog.io/treasuredata/api/gems/rubygems-local"
+    host = "https://rubygems.org"
```
Comment (dmikurube) [inline build.gradle:206]: As this plugin is open-sourced, we push this to public RubyGems.org.

**Review by ngochc — APPROVED** [review]
LGTM

**Review by hieudion — APPROVED** [review]
LGTM!

**Comment by dmikurube** [comment]
Thanks! I'll be releasing its v0.4.0 both as Gem and Maven.

**Pull request merged**